.github/workflows/ci.yaml

```diff
@@ -15,29 +15,44 @@ env:
   BUILD_TARGET: "x86_64-unknown-linux-gnu"
 
 jobs:
-  docker:
+  docker-targets-build:
     runs-on: ubuntu-latest
     if: github.event_name == 'push' && github.ref_type == 'tag'
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - platform: linux/amd64
+            target: x86_64-unknown-linux-gnu
+            build-config: ""
+          - platform: linux/arm64
+            target: aarch64-unknown-linux-gnu
+            build-config: "--config \"target.aarch64-unknown-linux-gnu.linker = 'aarch64-linux-gnu-gcc'\""
+    name: docker-targets-build (${{ matrix.platform.platform }})
     steps:
       - name: Checkout
         uses: actions/checkout@v3
         with:
           submodules: recursive
           fetch-depth: '0'
+      - name: Prepare
+        run: |
+          platform=${{ matrix.platform.platform }}
+          echo "PLATFORM_PAIR=${platform//\//-}" >> $GITHUB_ENV
       - name: Setup Rust
         uses: actions-rs/toolchain@v1
         with:
           toolchain: stable
+          target: ${{ matrix.platform.target }}
           override: true
-      - name: Install Rust Targets
-        run: |
-          rustup target add x86_64-unknown-linux-gnu
-          rustup target add aarch64-unknown-linux-gnu
+      - name: Install Rust Target
+        run: rustup target add ${{ matrix.platform.target }}
       - name: Install Protoc
         uses: arduino/setup-protoc@v1
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
       - name: Install gcc-aarch64-linux-gnu
+        if: matrix.platform.platform == 'linux/arm64'
```

Comment:
This indicates the problem of reuse.

Reply:
well, it is like that as it is build is made under adm64 linux, i can remove condition, it will be installed always